chrome/fed-modules.json

```diff
@@ -51,7 +51,7 @@
                 "id": "user-access",
                 "module": "./RootApp",
                 "routes": [
-                    "/settings/rbac"
+                    "/iam/user-access"
```

Comment:
Can we keep both routes for now? This change will mess up all QE tests that are currently in place.